Make draft email when email is clicked

leonyangela opened the pull request:
---
name: Pull Request 
about: When the user clicked support email, it will opens up mail app and create a draft email.
title: 'Make Chingu Support email clickable'
labels: 'Enhancement, good-first-issue, Front end'
assignees: 'leony'

---

When the user clicked support email, it will opens up mail app and create a draft email.

Resolves #198 

![chingu-email-draft-feature](https://user-images.githubusercontent.com/38250310/136524175-87ee9c92-cefb-44d7-a55f-0e1f37d8a1c3.png)

leonyangela committed:
Make draft email when email is clicked

When the user clicked support email, it will opens up a mail app and create a draft email.

Resolves #198

Soreg commented on pages/data-privacy.tsx:
```diff
@@ -39,7 +45,7 @@ export default function DataPrivacy() {
         <br />
 
-        <p>If you you have questions about this privacy policy contact us at support@chingu.io</p>
+        <p>If you you have questions about this privacy policy contact us at <EmailTo as="a" href="mailto:support@chingu.io">support@chingu.io</EmailTo></p>
```
It's probably easier to just use an a-tag directly (`<a href="..."`) - so no new styled component needed.
We have default styles for all our a-tags, so we don't need to add custom hover-styles 🙂

leonyangela replied:
sure, will fix that

leonyangela committed:
Change EmailTo styled component to a link

When the user clicked support email, it will opens up mail app and create a draft email.

Resolves #198

leonyangela committed:
Revert "Change EmailTo styled component to a link"

This reverts commit fc08df0b257bb6e6bcb1c5d638752203051d2700.

leonyangela committed:
Change EmailTo styled component to a link

When the user clicked support email, it will opens up mail app and create a draft email.

Resolves #198

Soreg commented on pages/data-privacy.tsx:
```diff
@@ -19,6 +19,7 @@ const Section = styled.div`
   }
 `;
 
+
```
Last bit of change: I see an extra line was added here (after the styled component was removed). Please remove this one - then it's fine 👍

leonyangela committed:
remove white space at the end

leonyangela committed:
Revert "remove white space at the end"

This reverts commit 80dbd44100b5461446015b45e98485cc6f5f2182.

leonyangela committed:
remove white space at line 22

Soreg reviewed (approved):
Approved - good job 👍

cMikolai reviewed (approved):
Looks good!
